### PR TITLE
Avatar navbar photo update

### PR DESCRIPTION
**Pull request by cawoolf**
Update so that user can change profile picture, and all animal cards have correct avatar!

### CODE REVIEW

File: app/controllers/pages_controller.rb [hunk 1]
```diff
@@ -1,3 +1,5 @@
+require "open-uri"
```
Comment (lauraylchen) [inline app/controllers/pages_controller.rb]: Remove it please

File: app/views/pages/user_profile.html.erb [hunk 2]
```diff
@@ -1,2 +1,18 @@
 <h1> User Profile </h1>
-<p> Upload a user profile </p>
\ No newline at end of file
+<p> Update a user profile </p>
+<% if current_user.photo.attached? %>
```
Comment (lauraylchen) [inline app/views/pages/user_profile.html.erb]: Remove this and render the current user in share folder

File: app/views/shared/_navbar.html.erb [hunk 3]
```diff
@@ -23,7 +23,13 @@
             <%= link_to "Post Animal", new_animal_path, class: "nav-link btn btn-lightblue mx-2" %>
           </li>
           <li class="nav-item dropdown">
-            <%= image_tag "professionalpicture.jpg", class: "avatar dropdown-toggle", id: "navbarDropdown", data: { bs_toggle: "dropdown" }, 'aria-haspopup': true, 'aria-expanded': false %>
+          
+            <% if current_user.photo.attached? %>
```
Comment (lauraylchen) [inline app/views/shared/_navbar.html.erb]: render this too

**Review by lauraylchen — COMMENTED** [review]
Please review this :)

File: app/views/shared/_navbar.html.erb (hunk 3 above)
Comment (cawoolf) [reply app/views/shared/_navbar.html.erb]: Rendered! But needed its own shared html to function as a drop down.

File: app/views/pages/user_profile.html.erb (hunk 2 above)
Comment (cawoolf) [reply app/views/pages/user_profile.html.erb]: Done! Rendered but allowed for the photo to be larger than just an avatar.

File: app/controllers/pages_controller.rb (hunk 1 above)
Comment (cawoolf) [reply app/controllers/pages_controller.rb]: Done!

File: app/views/shared/_navbar.html.erb (hunk 3 above)
Comment (cawoolf) [reply app/views/shared/_navbar.html.erb]: We could try to refactor all these avatar_html's into a single one, but for right now they all sort of have different uses/ styling.

**Review by Cyme101 — APPROVED** [review]
All good!

**Pull request merged**